pkg/apis/provisioning/v1alpha5/requirements.go

```diff
@@ -128,12 +135,12 @@ func (r Requirements) Label(key string) string {
 	return values.Values().UnsortedList()[0]
 }
 
-// Get returns the sets of values allowed by all included requirements
-// following a denylist method. Values are allowed except specified
+func (r Requirements) Has(key string) bool {
+	_, ok := r.requirements[key]
+	return ok
+}
+
 func (r Requirements) Get(key string) sets.Set {
-	if _, ok := r.requirements[key]; !ok {
```

Comment:
We no longer model nonexistance as the full set. The empty set and undefined set are both equivalent to {}.